src/drive/web/modules/queries.js

```diff
@@ -325,6 +325,37 @@ export const buildEncryptionByIdQuery = id => ({
   }
 })
 
+/**
+ * Provide selector and options to fetch files
+ *
+ * @returns {{options: {MangoQueryOptions}, selector: object}} A minimalist file list
+ */
+export const prepareSuggestionQuery = () => {
+  const selector = {
+    _id: {
+      $gt: null
+    }
+  }
```

Comment:
@trollepierre after the merge of https://github.com/cozy/cozy-client/pull/1225 you should be able to remove this selector

Reply:
done